perl.h

```diff
@@ -5723,6 +5723,12 @@ EXTCONST char PL_bincompat_options[] =
 #  ifdef DEBUG_LEAKING_SCALARS_FORK_DUMP
                              " DEBUG_LEAKING_SCALARS_FORK_DUMP"
 #  endif
+#  ifdef HAS_LONG_DOUBLE
+                             " HAS_LONG_DOUBLE"
+#  endif
+#  ifdef HAS_STRTOLD
```

Comment:
I think `HAS_STRTOLD` belongs in `non_bincompat_options[]` in perl.c, I don't see how it has an effect on binary compatibility.

That said, some of the existing `non_bincompat_options` probably belong in `PL_bincompat_options`.

Reply:
Ok, ill take a look.